lib/web/imageRouter.js

```diff
@@ -73,6 +73,40 @@ imageRouter.post('/uploadimage', function (req, res) {
               })
             })
             break
+
+          case 'minio':
+            var utils = require('../utils')
+            var Minio = require('minio')
+            var minioClient = new Minio.Client({
+              endPoint: config.minio.endPoint,
+              port: config.minio.port,
+              secure: config.minio.secure,
+              accessKey: config.minio.accessKey,
+              secretKey: config.minio.secretKey
+            });
```

Comment:
Please remove the unneeded semicolon to fix the linter warning: https://travis-ci.org/hackmdio/hackmd/jobs/267696722#L486

Reply:
Fixed